examples/vnet/doca_flow.c

```diff
@@ -38,12 +38,11 @@ struct doca_flow_pipe_entry *doca_flow_pipe_add_entry(
 	struct doca_flow_monitor *mon, struct doca_flow_fwd *fwd,
 	struct doca_flow_error *err)
 {
-	if (pipe == NULL || match == NULL || actions == NULL || mon == NULL)
+	if (pipe == NULL || match == NULL || actions == NULL)
 		return NULL;
 	pipe_queue = pipe_queue;
-	return doca_dpdk_pipe_create_flow(pipe, pipe_queue, match, actions, mon,
-					  (fwd != NULL)?fwd:&pipe->fwd,
-                                           err);
+	return doca_dpdk_pipe_create_flow(pipe, pipe_queue, match, actions, NULL,
```

Comment:
you removed the mon

Reply:
will revert

Reply:
but what if the user doesn't provide a mon then we will fail to add the entry, i think mon should be treated as fwd this is why i removed it from the if, what do you think ?